index.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="kr">
```

Comment:
요거 조금 헷갈리죠 ㅎㅎ kr이 아니라 ko로 변경해주셔야 올바른 문법입니다!

```suggestion
<html lang="ko">
```

Reply:
이거 기억하도록 하겠습니다. 감사합니다.